ulozto_downloader/downloader.py

```diff
@@ -39,17 +39,19 @@ def terminate(self):
         print('Download terminated.')
         return
 
-    def _print(self, text, x=0, y=0):
+    @staticmethod
+    def _print(text, x=0, y=0):
         sys.stdout.write("\033[{};{}H".format(y, x))
         sys.stdout.write("\033[K")
         sys.stdout.write(text)
         sys.stdout.flush()
 
-    def _print_part_status(self, id, text):
-        self._print(colors.blue(f"[Part {id}]") + f"\t{text}", y=(id + CLI_STATUS_STARTLINE))
+    @staticmethod
+    def _print_part_status(id, text):
+        Downloader._print(colors.blue(f"[Part {id}]") + f"\t{text}", y=(id + CLI_STATUS_STARTLINE))
 
     def _print_captcha_status(self, text):
```

Comment:
U `_print_captcha_status` není potřeba `@staticmethod`? Tahle funkce se předává do odděleného procesu pro louskání CAPTCHA, takže bych tam čekal stejný problém...

Reply:
@setnicka Jo, to je zvláštní. Tak možná to není tím, já nevím. Rozumnou chybovou hlášku to nepsalo, nic se mi nepodařilo najít. #itJustWorks.